.gitattributes

```diff
@@ -0,0 +1,26 @@
+# Auto detect text files and perform LF normalization
+* text eol=lf
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+
+# files part of the build
+*.asm text
+*.pal text
+*.link text
+
+# extra files
+*.c text
+*.h text
+*.md text
+*.md text
```

Comment:
duplicate line here

Reply:
And this is why we have reviewers. Nice catch lol.

Reply:
The duplicate line has been removed.